packages/components/src/button/README.md

```diff
@@ -6,9 +6,18 @@ Buttons let users take actions and make choices with a single click or tap.
 
 ## Table of contents
 
-1. [Design guidelines](#design-guidelines)
-2. [Development guidelines](#development-guidelines)
-3. [Related components](#related-components)
+1. [Storybook tool](#storybook-tool)
```

Comment:
I'm not 100% sure that the Storybook link should be first on the list. I think perhaps the user should see the guidelines before they can the link. I'm not totally sure but that was my first thought.